File: backend/tildau/src/main/java/kz/kbtu/tildau/dto/exercise/ExerciseFullResponse.java
package kz.kbtu.tildau.dto.exercise;

import kz.kbtu.tildau.enums.ExerciseType;
import lombok.*;

import java.util.UUID;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Getter
@Builder
public class ExerciseFullResponse {
    private UUID id;
    private String title;
    private String instruction;
    private ExerciseType exerciseType;
    private String expectedText;
    private String referenceAudioUrl;


}
 

File: android/Tildau2/app/src/main/java/com/example/tildau/data/model/exercise/ExerciseFullResponse.kt
package com.example.tildau.data.model.exercise

import java.io.Serializable

data class ExerciseFullResponse(
    val id: String,
    val title: String,
    val instruction: String,
    val exerciseType: ExerciseType,
    val expectedText: String?,
    val referenceAudioUrl: String?
) : Serializable